pkg/mcs/tso/server/server.go

```diff
@@ -389,83 +394,128 @@ func (s *Server) startGRPCServer(l net.Listener) {
 	defer logutil.LogPanic()
 	defer s.serverLoopWg.Done()
 
-	gs := grpc.NewServer()
-	s.service.RegisterGRPCService(gs)
-	diagnosticspb.RegisterDiagnosticsServer(gs, s)
-	serverr := gs.Serve(l)
-	log.Info("grpc server stopped serving")
-
-	// Attempt graceful stop (waits for pending RPCs), but force a stop if
-	// it doesn't happen in a reasonable amount of time.
-	done := make(chan struct{})
-	go func() {
-		defer logutil.LogPanic()
-		log.Info("try to gracefully stop the server now")
-		gs.GracefulStop()
-		close(done)
-	}()
-	select {
-	case <-done:
-	case <-time.After(mcsutils.DefaultGRPCGracefulStopTimeout):
-		log.Info("stopping grpc gracefully is taking longer than expected and force stopping now")
-		gs.Stop()
-	}
-
+	log.Info("[tso] grpc server starts serving", zap.String("address", l.Addr().String()))
+	err := s.grpcServer.Serve(l)
 	if s.IsClosed() {
-		log.Info("grpc server stopped")
+		log.Info("[tso] grpc server stopped")
 	} else {
-		log.Fatal("grpc server stopped unexpectedly", errs.ZapError(serverr))
+		log.Fatal("[tso] grpc server stopped unexpectedly", errs.ZapError(err))
```

Comment:
We don't need to change it because [xxx] is mainly used for the client.

Reply:
removed all